src/main/java/frc/robot/RobotContainer.java

```diff
@@ -33,16 +36,20 @@ public class RobotContainer {
   private final SimulationSubsystem simulationSubsystem;
   private final EverybotIntakeSubsystem intakeSubsystem;
   private final CameraSubsystem cameraSubsystem;
+  private LedSubsystem ledSubsystem;
+  private final LedMatrixSubsystem ledMatrixSubsystem;
 
   private Auton auton;
 
   private AssistedBalanceCommand assistedBalanceCommand;
 
   private DataLog robotSubsystemsLog = DataLogManager.getLog();
-  private StringLogEntry subsystemEnabledLog = new StringLogEntry(robotSubsystemsLog, "/Subsystems Enabled/");
-
+  private StringLogEntry subsystemEnabledLog = new StringLogEntry(robotSubsystemsLog, "/Subsystems Enabled/"); 
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
+    //ledSubsystem = new LedSubsystem(); //PWM port 1
+    ledMatrixSubsystem = new LedMatrixSubsystem(); //PWM port 2
```

Comment:
Since we have a few options to select from for our LED Subsystem, creating a Java Interface would be useful so that we can more easily swap between them.

This should be a future iteration though.

Reply:
Added a card to trello, closing the comment